dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hive/HiveMetadataConnector.java

```diff
@@ -135,6 +138,39 @@ protected String toCallDescription() {
     }
   }
 
+  private static class DatabasesJsonlTask extends AbstractHiveMetadataTask {
+
+    private DatabasesJsonlTask() {
+      super("databases.jsonl", unused -> true);
```

Comment:
The 2nd parameter equivalent to `Predicates.alwaysTrue` shouldn't be necessary here.
My suggestion is to either add an alternative `AbstractHiveMetadataTask` constructor which doesn't set `predicate` or, preferably, add a trivial `isIncludedDatabase` implementation to `AbstractHiveTask` and inherit from it directly where possible.

Reply:
I've added a second constructor. Filtering by database makes sense only in the context of metadata.

However, currently there are no other classes extending `AbstractHiveTask` besides `AbstractHiveMetadataTask`, so possibly these two could be merged.